src/models/annotation.model.js

```diff
@@ -0,0 +1,78 @@
+'use strict'
+
+const DynamicsDalService = require('../services/dynamicsDal.service')
+const BaseModel = require('./base.model')
+const LoggingService = require('../services/logging.service')
+const Utilities = require('../utilities/utilities')
+
+module.exports = class Annotation extends BaseModel {
+  constructor (annotation) {
+    super()
+    this.entity = 'annotations'
+    this.id = annotation.id
+    this.subject = annotation.subject
+    this.filename = annotation.filename
+    this.documentBody = annotation.documentBody
+    this.applicationId = annotation.applicationId
+    Utilities.convertFromDynamics(this)
+  }
+
+  static selectedDynamicsFields () {
```

Comment:
I like the 'selectedDynamicsFields' approach - I will refactor our other model classes to match.